pfcpiface/utils.go

```diff
@@ -11,6 +11,19 @@ import (
 	"strings"
 )
 
+type Bits uint8
+
+const (
+	FwdIE_OuterHeaderCreation Bits = 1 << iota
+	FwdIE_DestinationIntf
+)
```

Comment:
Should be within respective files. In this case parse-far.go.

Reply:
done